test/common/test_io_elements.py

```diff
@@ -190,6 +190,10 @@ def test_page():
     assert page.export() == {"blocks": [b.export() for b in blocks], "page_idx": page_idx, "dimensions": page_size,
                              "orientation": orientation, "language": language}
 
+    # Export XML
+    assert isinstance(page.export_as_xml(), tuple) and isinstance(
+        page.export_as_xml()[0], (bytes, bytearray))
```

Comment:
We should check the type of the second returned objects :sweat_smile:

Reply:
@fg-mindee 
yes, I have to investigate a simple `isinstance(page.export_as_xml()[1], ElementTree)` doesnt work